.github/workflows/main.yml

```diff
@@ -24,62 +24,62 @@ jobs:
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
     steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: recursive
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
```

Comment:
can we please revert the formatting changes and only keep the wasm one? just asking for making it easier for resync with upstream